### ISSUE TITLE
Clarifying on how to setup the stoploss_on_exchange

### ISSUE THREAD

**Issue by geohuz** [freqtrade/freqtrade#11208]
## Your question

* I'm not sure how to set a stoploss_on_exchange, my configuration is like this:

```python
"order_types": {
    "entry": "limit",                 
    "exit": "limit",                  
    "stoploss": "stoploss_on_exchange"  
},
```

As stated in doc, "If stoploss_on_exchange uses limit orders, the exchange needs 2 prices, the stoploss_price and the Limit price.", I don't know how to set the stoploss_price and limit price, there is no example in doc for this, So I'm guessing this config:

```
stoploss_price: (a fixed price)
````

Basically I want to set a stoploss price to h2 open at time of entry, whereas it could be a percentage or a fixed value. My first attemp is as this:

```python
class MyStrategy(IStrategy):
    # Define parameters
    stoploss_on_exchange = True
    stoploss_limit_ratio = 0.99

    # calculate my actual stoploss percentage on trade_entry
    def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: float,
                            time_in_force: str, side: str, **kwargs) -> bool:

        dataframe = self.dp.get_pair_dataframe(pair=pair, timeframe=self.timeframe)

        # Get the trade entry point
        trade = kwargs.get('trade')
        entry_index = dataframe.index.get_loc(trade.open_date_utc)
        
        # Calculate the previous H2 at the entry point (highest high of the last 2 candles before entry)
        if entry_index >= 2:
            previous_h2 = dataframe['high'].iloc[entry_index-2:entry_index].max()
        else:
            previous_h2 = dataframe['high'].iloc[:entry_index].max()  # Handle cases where entry is too early in the data
        
        # Store the stop-loss as a percentage relative to the trade open rate
        self.stop_loss = (previous_h2 - trade.open_rate) / trade.open_rate

        return True

```

Here I set the stoploss to a value based on the last h2 price at the time of entry. Not sure if it works

**Comment by xmatthias** [comment]
Your configuration is (absolutely) incorrect.


Please refer to the [documentation](https://www.freqtrade.io/en/stable/stoploss/#stop-loss-on-exchangefreqtrade) about this topic on the proper configuration. It contains the necessary settings - as well as how to define the "2nd" price (which you'll define as a ratio of the first value statically).

Assigning `self.stop_loss` is also not a supported approach for this and can have unintended side-effects.
If you need dynamic stoploss - there's a [callback](https://www.freqtrade.io/en/stable/strategy-callbacks/#custom-stoploss) for that.
